src/sentry/api/endpoints/group_similar_issues_embeddings.py

```diff
@@ -50,13 +58,49 @@ def get_stacktrace_string(exception: Mapping[Any, Any], event: GroupEvent) -> st
     return "\n".join(output)
 
 
+class FormattedSimilarIssuesEmbeddingsData(TypedDict, total=False):
+    exception: float
+    message: float
+    shouldBeGrouped: str
+
+
 @region_silo_endpoint
 class GroupSimilarIssuesEmbeddingsEndpoint(GroupEndpoint):
     owner = ApiOwner.ISSUES
     publish_status = {
         "GET": ApiPublishStatus.PRIVATE,
     }
 
+    def get_formatted_results(
+        self, responses: Sequence[SimilarIssuesEmbeddingsData | None], user: User | AnonymousUser
+    ) -> Sequence[tuple[Mapping[str, Any], Mapping[str, Any]] | None]:
+        """Format the responses using to be used by the frontend."""
+        group_data = {}
+        for response in responses:
+            if response:
```

Comment:
Can `response` be an empty element or None? Is that why you have the `if` clause?
Do you test for it? (since I see conditional coverage for this line).